extensions/wikia/AdEngine/js/AdConfigMobile.js

```diff
@@ -28,6 +29,10 @@ define('ext.wikia.adEngine.adConfigMobile', [
 			return [];
 		}
 
+		if (context.forceProviders.openX) {
```

Comment:
Important player since it's the first `if` now returning a provider :japanese_ogre: